Selenium.Axe/Selenium.Axe/AxeResultTarget.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Selenium.Axe
+{
+    public class AxeResultTarget
+    {
+        public string Selector { get; set; }
```

Comment:
This is confusing enough to use correctly that it's probably a good idea to leave some comments on the class and properties explaining that "Selector" and "Selectors" are mutually exclusive and when you'd see one vs the other; if I were a user of the library not familiar with Axe, I would be looking at this and thinking to myself "This is confusing. What is the difference between "Selector" and "Selectors"? Why would I look at one instead of the other?"

Reply:
Good point.  Comment added.